hack/keda_values.yaml

```diff
@@ -0,0 +1,45 @@
+global:
+  image:
+    registry: "europe-docker.pkg.dev/kyma-project/prod/external/ghcr.io"
+
+prometheus:
+  metricServer:
+    # -- Enable metric server Prometheus metrics expose
+    enabled: true
+    # -- HTTP port used for exposing metrics server prometheus metrics
+    port: 8080
+    # -- HTTP port name for exposing metrics server prometheus metrics
+    portName: metrics
```

Comment:
why here it is defined but not for the other components? Maybe there is already a good default?
If you overwrite it, then please follow the conventions by using "http-metrics"

Reply:
because the `portName` is defined and used only for the metricServer component. more details [here](https://github.com/kedacore/charts/blob/release/v2.16/keda/values.yaml#L607)